package.json

```diff
@@ -45,7 +45,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "rebuild-db-online": "cross-env MISSING_MEDIA=link node preprocessing/rebuild.js",
     "rebuild-db-offline": "cross-env MISSING_MEDIA=ignore node preprocessing/rebuild.js",
-    "webpack": "webpack-cli --config webpack.config.js",
```

Comment:
This change here ("config" -> "conf") likely is the reason deployment of this pull request failed. Let's revert this change by creating a new pull request and manually change "conf" back to "config"